### ISSUE TITLE
After restarting my server I can't open my azuracast.

### ISSUE THREAD

**Issue by rolradio** [AzuraCast/AzuraCast#6173]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

#25236df (2022-12-27 9:17)

### What happened?

Hi,

After the data center had to stop the server and then start it again, I can no longer log in to my Azuracast.
When I try the link https://sub.domain.com or https://sub.domain.com/login or whatever I try, it keeps redirecting to https://sub.domain.com/index.html? sid=1
And I hear the sound of one of the stations on my azuracast.

docker-compose down and docker-compose up -d.
and even an update didn't fix it.
What could this be.

It seems to be a bug in nginx redirection
If I use http with the subdomain or the IP address nothing works.

I also send the update log, everything seems to be normal here.
Please help because I don't want to reinstall everything again.

### Relevant log output

```shell
I also send the update log, everything seems to be normal here.
I think.

root@vmi931897:~# cd /var/azuracast
root@vmi931897:/var/azuracast# ./docker.sh update-self
New Docker utility script downloaded.
root@vmi931897:/var/azuracast# ./docker.sh update
[NOTICE] Before you continue, please make sure you have a recent snapshot of your system and or backed it up.
Are you ready to continue with the update? [Y/n] Y
Docker server (version 23.0.1) meets minimum version requirements.
[+] Running 1/1
 ⠿ installer Pulled                                                                            0.5s
[+] Running 1/1
 ⠿ Network azuracast_installer_default  Created                                                0.1s

AzuraCast Updater
=================

 Change installation settings? (yes/no) [no]:
 > no

Writing configuration files...


 [OK] Server configuration complete!


[+] Running 2/2
 ⠿ updater Pulled                                                                              0.5s
 ⠿ web Pulled                                                                                  0.5s
[+] Running 3/3
 ⠿ Container azuracast_updater  Removed                                                        0.2s
 ⠿ Container azuracast          Removed                                                       14.4s
 ⠿ Network azuracast_default    Removed                                                        0.1s
[+] Running 1/0
 ⠿ Network azuracast_default  Created                                                          0.1s
** Running startup script '/etc/my_init.d/00_disable_mariadb.sh'...
** Startup script complete.
** Running startup script '/etc/my_init.d/00_disable_redis.sh'...
** Startup script complete.
** Running startup script '/etc/my_init.d/00_setup_user.sh'...
usermod: no changes
Docker 'azuracast' User UID: 1000
Docker 'azuracast' User GID: 1000
** Startup script complete.
** Running startup script '/etc/my_init.d/01_self_signed_ssl.sh'...
Generating self-signed certificate...
....+...+...+...+.+...+...........+......+....+...............+.....+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.....+....+........+..........+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.......+......+...+.+..+............+.+..+.......+.....+.+......+...+..+...............+......+...+...................+...+...+............+..............+.............+...+..+.+.....+....+..+...+..................+...............+.........+....+..+...+.......+...+.................+....+...+......+.........+...+.......................+.+...+...+..+................+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
.....+........+...+...+.+.....+.+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.....+..+.+......+.....+....+.........+...+.....+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*...+..+...+..........+.....+.+.........+.....+.............+...+.....+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-----
** Startup script complete.
** Running startup script '/etc/my_init.d/04_mariadb_conf.sh'...
** Startup script complete.
** Running startup script '/etc/my_init.d/05_centrifugo_conf.sh'...
** Startup script complete.
** Running startup script '/etc/my_init.d/05_nginx_conf.sh'...
** Startup script complete.
** Running startup script '/etc/my_init.d/05_setup_db.sh'...
2023-03-23 21:49:36+00:00 [Note] [Entrypoint]: Initial DB setup...
2023-03-23 21:49:36+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
2023-03-23 21:49:37+00:00 [Note] [Entrypoint]: Initial DB setup...
2023-03-23 21:49:37+00:00 [Note] [Entrypoint]: MariaDB upgrade not required
** Startup script complete.
** Running startup script '/etc/my_init.d/07_sftpgo_conf.sh'...
** Startup script complete.
** Running startup script '/etc/my_init.d/08_disable_meilisearch.sh'...
** Startup script complete.
** Running startup script '/etc/my_init.d/10_clear_temp.sh'...
** Startup script complete.
** Running Supervisord...
** Supervisord started.
** Running uptime_wait...
Starting up AzuraCast services...
🖥️－🎶－🎵📻 Waiting...Services started up and ready!
** Services up and ready!
This script must be run as the "azuracast" user. Rerunning...
AzuraCast Setup
(Environment: production)

AzuraCast Setup
===============

Welcome to AzuraCast. Please wait while some key dependencies of AzuraCast are set up...

Running Database Migrations
---------------------------


Database Migrations
===================



 [OK] Metadata storage synchronized



 [OK] Database is already up to date!



Generating Database Proxy Classes
---------------------------------


 Processing entity "App\Entity\Settings"
 Processing entity "App\Entity\SongHistory"
 Processing entity "App\Entity\StorageLocation"
 Processing entity "App\Entity\StationMedia"
 Processing entity "App\Entity\StationPlaylistFolder"
 Processing entity "App\Entity\SftpUser"
 Processing entity "App\Entity\AuditLog"
 Processing entity "App\Entity\Relay"
 Processing entity "App\Entity\StationRequest"
 Processing entity "App\Entity\StationMediaCustomField"
 Processing entity "App\Entity\CustomField"
 Processing entity "App\Entity\StationWebhook"
 Processing entity "App\Entity\Analytics"
 Processing entity "App\Entity\Station"
 Processing entity "App\Entity\StationHlsStream"
 Processing entity "App\Entity\Role"
 Processing entity "App\Entity\PodcastMedia"
 Processing entity "App\Entity\StationQueue"
 Processing entity "App\Entity\Listener"
 Processing entity "App\Entity\StationPlaylist"
 Processing entity "App\Entity\StationPlaylistMedia"
 Processing entity "App\Entity\RolePermission"
 Processing entity "App\Entity\StationStreamerBroadcast"
 Processing entity "App\Entity\Podcast"
 Processing entity "App\Entity\UnprocessableMedia"
 Processing entity "App\Entity\ApiKey"
 Processing entity "App\Entity\PodcastCategory"
 Processing entity "App\Entity\StationSchedule"
 Processing entity "App\Entity\StationMount"
 Processing entity "App\Entity\StationStreamer"
 Processing entity "App\Entity\PodcastEpisode"
 Processing entity "App\Entity\User"
 Processing entity "App\Entity\StationRemote"
 Processing entity "App\Entity\UserLoginToken"

 Proxy classes generated to "/var/azuracast/www_tmp/proxies"

Reload System Data
------------------



 [OK] Local cache flushed.



Refreshing All Stations
-----------------------



 [OK] Unplayed station queues cleared.



Restarting all radio stations...
--------------------------------
```

**Comment by BusterNeece** [comment]
@rolradio Did you somehow set one of your stations to operate on port 80 or 443? That would possibly cause that...and I'm not sure what the immediate solution to that would be, to be honest. If you set it to listen on port 80, you could try manually visiting `your-url:443/login` and fixing it from there...

**Comment by rolradio** [comment]
@BusterNeece     No there are no ports 443 or 80 in use by the radios.
It is a standard docker installation.
With a port extension from 7900.
So nothing special.

Unfortunately, nothing helps here.
ip address or domain:443 gives the same result.


I've had this before, but that was after an update.
It's strange that it happened after a server shutdown.

I'm afraid it will be another reinstall.

**Comment by BusterNeece** [comment]
@rolradio This is the first report I've ever seen of this, though, of a Shoutcast station "taking over" the main web ports on an installation unprompted. The only possible cause of this that I can think of is a misconfiguration of a station.

**Comment by rolradio** [comment]
@BusterNeece     Luckily it's resolved.
You got me started with the tip about port 443

There was a radio station that
"alternateports": "443" used in the shoutcast config.
I was able to remove this via ssh.
In the sc_serv.conf file of the affected radio.
And the file was given a different permission so that it couldn't be overwritten by restarting azuracast.

After restart everything works again, and I was able to remove the line "alternateports": "443" in the Custom configuration for broadcasting in azuracast.
Thanks for the tip.
